CRM/Oapproviderlistapp/Form/Search/ProviderList.php

```diff
@@ -218,7 +218,7 @@ public function sql(
       $sql .= " $groupBy ";
     }
 
-    $sort = 'contact_a.first_name, contact_a.last_name';
+    $sort = 'contact_a.last_name, contact_a.first_name';
```

Comment:
This should have been obvious.

Reply:
@JoeMurray Actually this fix was first made on 18th December and this was the [commit](https://github.com/JMAConsulting/biz.jmaconsulting.oapproviderlistapp/commit/eff2743e472593bd11a18ee30818d421bcfa44d2) in request to a task listed in 5th item - 'Please alphabetize result by title.' in https://jmaltd.atlassian.net/browse/OAPS-144 (your comment on December 16, 2019, 9:32 PM) At that point I wasn't sure about whether the title should be sorted on basis of first_name or last_name as core usually picks first_name by default for sort.  The reason why chose the default order for sort.